### PR TITLE
tools: setup GHA for PR notifier

### PR DESCRIPTION
**Pull request by mattklein123**

### CODE REVIEW

**Comment by mattklein123** [comment]
@alyssawilk I think it's pretty much as easy as this. Right now I have it running on PR just to test it, but then we can switch it over to both a cron and manual dispatch. I can also help you with the secrets offline.

**Comment by mattklein123** [comment]
cc @phlax also

File: .github/workflows/pr_notifier.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+          architecture: 'x64'
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r ./.github/actions/pr_notifier/requirements.txt
```
Comment (phlax) [inline .github/workflows/pr_notifier.yml:22]: +1 - the pip installer i linked doesnt support require-hashes afaict - this sees to be the most straightforward

**Comment by alyssawilk** [comment]
Nice!!

**Comment by daixiang0** [comment]
I am not sure whether it can be replaced by this [tool](https://slack.com/help/articles/232289568-GitHub-for-Slack#github-notifications-(legacy)-1).

**Comment by mattklein123** [comment]
I'm just going to close this. @alyssawilk this is simple enough let's just merge this into your other PR?

**Pull request closed without merging**